### DISCUSSION TITLE
Speed mode issue

### DISCUSSION THREAD

**Discussion by ghairapetian** [google/digitalbuildings#788] [Q&A] [answered]
Hello,

I have a warm air curtain with 6 differents speed mode (0, 20, 40, 60, 80, 100)%

The telemetry field discharge_fan_speed_mode mention the following :

AUTO
LOW
MEDIUM
HIGH
OFF
VFD
that does not correspond to my needs. However, in my case, it's a fan on discharge side, and it runs with speed mode. So I can't really create a new telemtry with a new name.

How can i resolve this issue ?

Thank you very much,

Have a nice day,

**Comment by tasodorff** [comment, marked as answer]
if this cannot be mapped as a continuous numeric value and modeled as a sensor (which is by far the preference) then an alternative would be to add descriptive states for the field. for example: https://github.com/google/digitalbuildings/blob/cb61b1e2d1ec06b4b619fe0f0400100bd42159fa/ontology/yaml/resources/states/states.yaml#L24
